### PR TITLE
David Loveday

### PR DESCRIPTION
**Pull request by AquilaVirtual**

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -7,71 +7,108 @@ describe('project-1', () => {
...
+        expect(helpers.lessThanNinety(100)).toBe(false);         
+      }); 
+   });
+ });
+
+ describe('project-1', () => {
+  describe('greaterThanFifty', () => {
+    it('should return a boolean to indicate a given number is greater than 50', () => {
+        expect(helpers.greaterThanFifty(70)).toBe(true);  
+        expect(helpers.greaterThanFifty(20)).toBe(false);
```
Comment (JasonCorp84) [inline basic-javascript/project-1.spec.js]: a nice edge case would be: test 50 and 51 also, so the developer did not use >= instead of >

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -82,24 +119,103 @@ describe('project-1', () => {
...
+        it('should return area of a rectangle by its length and width', () => {
+            expect(helpers.getRectangleArea(8, 5)).toBe(40);
+        });
+    });
+})
+
+describe('project-1', () => {
+    describe('getCircleArea', () => {
+        it('should return area of a circle by its radius', () => {
+            expect(helpers.getCircleArea(8)).toBeCloseTo(201, 0);
```
Comment (JasonCorp84) [inline basic-javascript/project-1.spec.js]: awesome!

**Comment by JasonCorp84** [comment]
David,

Try to use more messages and edge cases for the developer. 
I like that you used different tests (closetobe)
